src/lifecycle/props-init.js

```diff
@@ -14,14 +14,18 @@ function getInitialValue(elem, name, opts) {
   return typeof opts.initial === 'function' ? opts.initial(elem, { name }) : opts.initial;
 }
 
+function getPropNamespace(name) {
```

Comment:
You could abstract this further into a data function for namespaced props rather than having to repeat the `getPropNamespace()` call.